app/lib/shared/changelog.dart

```diff
@@ -401,6 +401,11 @@ class ChangelogParser {
       }
     }
 
+    // remove leading 'Version' label
+    if (inputLowerCase.startsWith('version ')) {
+      input = input.trim().substring(7).trim();
```

Comment:
Do you really need both `.trim`s?

Reply:
good point, as `input` is already trimmed when the method is being called, and we don't need trim for `inputLowerCase` creation either.